app/search_builders/curation_concerns/filter_suppressed_with_roles.rb

```diff
@@ -0,0 +1,28 @@
+module CurationConcerns
+  module FilterSuppressedWithRoles
+    extend ActiveSupport::Concern
+    include CurationConcerns::FilterSuppressed
+
+    def only_active_works(solr_parameters)
+      if current_work.respond_to?(:to_sipity_entity) # Used because Collections use WorkSearchBuilder to filter down to a single item
```

Comment:
Maybe the Collections shouldn't use WorkSearchBuilder. That's probably better than adding these conditionals.